lib/Request.js

```diff
@@ -110,8 +118,9 @@ Request.prototype.getParam = function(param) {
     return this._params.hasOwnProperty(param) ? this._params[param] : undefined;
 };
 
-Request.prototype.setQueries = function(queries) {
-    this._url.query = queries;
+Request.prototype.setQueries = function(query) {
```

Comment:
We're also gonna validate the queries object after we set it .... so it'll be verified in all cases.... there's a bunch of reasons why i didn't make it validate before hand... we can talk in person and I can explain